impl_core/src/test/java/io/opencensus/implcore/stats/MutableAggregationTest.java

```diff
@@ -135,6 +140,43 @@ public Void apply(MutableLastValue arg) {
     }
   }
 
+  @Test
+  public void testAdd_DistributionWithExemplarAttachments() {
+    MutableDistribution mutableDistribution = MutableDistribution.create(BUCKET_BOUNDARIES);
+    List<Double> values = Arrays.asList(-1.0, 1.0, -5.0, 20.0, 5.0);
+    @SuppressWarnings("unchecked")
+    List<Map<String, String>> attachmentsList =
+        Arrays.<Map<String, String>>asList(
```

Comment:
I think you could use `ImmutableList.<Map<String, String>>of` to avoid the "unchecked" warning, since the list doesn't need to be modified.

Reply:
I'm trying to avoid using Guava classes and methods - is it OK to use them in the impl?

Reply:
I think we should eventually remove the dependency on Guava from the impl, though I don't think we need to avoid using it in tests.

Reply:
SG, I updated to use `ImmutableList` instead, since we have been using Guava classes in tests anyway.